resources/views/bread/browse.blade.php

```diff
@@ -66,29 +66,29 @@
                                         @foreach($dataType->browseRows as $row)
-                                        <th>
-                                            @if ($isServerSide)
-                                                <a href="{{ $row->sortByUrl() }}">
-                                            @endif
-                                            {{ $row->display_name }}
-                                            @if ($isServerSide)
-                                                @if ($row->isCurrentSortField())
-                                                    @if (!isset($_GET['sort_order']) || $_GET['sort_order'] == 'asc')
-                                                        <i class="voyager-angle-up pull-right"></i>
-                                                    @else
-                                                        <i class="voyager-angle-down pull-right"></i>
-                                                    @endif
+                                            <th>
+                                                @if ($isServerSide)
+                                                    <a href="{{ $row->sortByUrl() }}">
+                                                        @endif
+                                                        {{ $row->display_name }}
+                                                        @if ($isServerSide)
+                                                            @if ($row->isCurrentSortField())
+                                                                @if (!isset($_GET['sort_order']) || $_GET['sort_order'] == 'asc')
+                                                                    <i class="voyager-angle-up pull-right"></i>
+                                                                @else
+                                                                    <i class="voyager-angle-down pull-right"></i>
+                                                                @endif
+                                                            @endif
+                                                    </a>
                                                 @endif
-                                                </a>
-                                            @endif
-                                        </th>
+                                            </th>
                                         @endforeach
                                         <th class="actions text-right">{{ __('voyager::generic.actions') }}</th>
                                     </tr>
                                 </thead>
                                 <tbody>
-                                    @foreach($dataTypeContent as $data)
-                                    <tr>
+                                @foreach($dataTypeContent as $data)
+                                    <tr class="js-row" data-id="{{ $data->getKey() }}" >
```

Comment:
Is `js-row` a name you came up with? If so, it's far too generic.  Please come up with a more specific name.